server/conf/typedb.yml

```diff
@@ -16,8 +16,17 @@
 #
 
 
-data: server/data
-port: 1729
+server:
+  address: 0.0.0.0:1729
+
+storage:
+  data: server/data
+  db-cache:
+    # configure data and index per database in the storage layer
+    # for large datasets, it is more important to have a large index cache than a data cache
+    data: 500mb
+    index: 500mb
```

Comment:
refactored configurations - `address` contains both a listen address and a port
`storage` contains the `data` directory and the per-database caches